devel/howdoi/Portfile

```diff
@@ -2,35 +2,28 @@
 
 PortSystem          1.0
 PortGroup           python 1.0
+PortGroup           github 1.0
 
+github.setup        gleitz howdoi 2.0.19 v
```

Comment:
please do not switch to GitHub for Python ports if there is no reason to do so; PyPI is the preferred source

Reply:
Ah, you want it to stay with `pypi` repo…

Reply:
@reneeotten Done.